service/src/main/java/bio/terra/pipelines/stairway/imputation/gcp/SubmitCromwellRunSetStep.java

```diff
@@ -0,0 +1,99 @@
+import bio.terra.pipelines.dependencies.stairway.JobMapKeys;
+import bio.terra.pipelines.stairway.imputation.RunImputationJobFlightMapKeys;
+import bio.terra.rawls.model.SubmissionReport;
+import bio.terra.rawls.model.SubmissionRequest;
+import bio.terra.stairway.*;
+
+/**
+ * This step submits a run set to cromwell using the rawls submission endpoint.
+ *
+ * <p>this step expects nothing from the working map
+ *
+ * <p>this step writes submission_id to the working map
+ */
+public class SubmitCromwellRunSetStep implements Step {
+  private final SamService samService;
+  private final RawlsService rawlsService;
+  private final ImputationConfiguration imputationConfiguration;
+
+  public SubmitCromwellRunSetStep(
+      RawlsService rawlsService,
+      SamService samService,
+      ImputationConfiguration imputationConfiguration) {
+    this.samService = samService;
+    this.rawlsService = rawlsService;
+    this.imputationConfiguration = imputationConfiguration;
+  }
+
+  @Override
+  @SuppressWarnings(
+      "java:S2259") // suppress warning for possible NPE when calling pipelineName.getValue(),
+  //  since we do validate that pipelineName is not null in `validateRequiredEntries`
+  public StepResult doStep(FlightContext flightContext) {
+    // validate and extract parameters from input map
+    FlightMap inputParameters = flightContext.getInputParameters();
+    FlightUtils.validateRequiredEntries(
+        inputParameters,
+        JobMapKeys.PIPELINE_NAME.getKeyName(),
+        RunImputationJobFlightMapKeys.CONTROL_WORKSPACE_PROJECT,
+        RunImputationJobFlightMapKeys.CONTROL_WORKSPACE_NAME);
+
+    String controlWorkspaceName =
+        inputParameters.get(RunImputationJobFlightMapKeys.CONTROL_WORKSPACE_NAME, String.class);
+    String controlWorkspaceProject =
+        inputParameters.get(RunImputationJobFlightMapKeys.CONTROL_WORKSPACE_PROJECT, String.class);
+    PipelinesEnum pipelineName =
+        inputParameters.get(JobMapKeys.PIPELINE_NAME.getKeyName(), PipelinesEnum.class);
+
+    // validate and extract parameters from working map
+    FlightMap workingMap = flightContext.getWorkingMap();
+
+    // create submission request
+    SubmissionRequest submissionRequest =
+        new SubmissionRequest()
+            .entityName(flightContext.getFlightId())
+            .entityType(
+                pipelineName.getValue()) // this must match the configuration the method is set to
+            // launch with.  Do we want to modify the current method
+            // configuration each time so this matches?
+            .useCallCache(true)
+            .deleteIntermediateOutputFiles(false)
+            .useReferenceDisks(false)
+            .userComment("this is my flightid: %s".formatted(flightContext.getFlightId()))
+            .methodConfigurationNamespace(controlWorkspaceProject)
+            .methodConfigurationName(
+                "ImputationBeagleEmpty"); // need to not hardcode this eventually
```

Comment:
are we able to import a WDL into a workspace with a custom name like we did in Azure? so like get the ImputationBeagleEmpty wdl but have it be called ImputationBeagle? I think this is possible (we'd need to create a separate workspace for it, like we did in Azure) and then we could have this come from the db like before